frontend/src/employee-frontend/components/unit/unit-reservations/StaffAttendanceTable.tsx

```diff
@@ -564,14 +574,33 @@ const AttendanceRow = React.memo(function AttendanceRow({
                 <AttendanceTimes data-qa="attendance-day">
                   {attendancesForToday.length > 0 ? (
                     attendancesForToday.map((attendance, i) => (
-                      <AttendanceCell key={i}>
-                        <AttendanceTime data-qa="arrival-time">
-                          {renderTime(attendance.arrived, date)}
-                        </AttendanceTime>
-                        <AttendanceTime data-qa="departure-time">
-                          {renderTime(attendance.departed, date)}
-                        </AttendanceTime>
-                      </AttendanceCell>
+                      <>
+                        {attendance.departedAutomatically ? (
+                          <Tooltip
+                            key={i}
+                            tooltip={attendanceTooltipText(attendance)}
+                            data-qa="attendance-tooltip"
+                          >
+                            <AttendanceCell>
+                              <AttendanceTime data-qa="arrival-time">
+                                {renderTime(attendance.arrived, date)}
+                              </AttendanceTime>
+                              <AttendanceTime data-qa="departure-time">
+                                {renderTime(attendance.departed, date)} (*)
+                              </AttendanceTime>
+                            </AttendanceCell>
+                          </Tooltip>
+                        ) : (
+                          <AttendanceCell>
+                            <AttendanceTime data-qa="arrival-time">
+                              {renderTime(attendance.arrived, date)}
+                            </AttendanceTime>
+                            <AttendanceTime data-qa="departure-time">
+                              {renderTime(attendance.departed, date)}
+                            </AttendanceTime>
+                          </AttendanceCell>
+                        )}
+                      </>
```

Comment:
I think duplicating the whole thing is unnecessary. AFAIK tooltip is not rendered if tooltip prop is falsy.

```
const attendanceTooltip = attendanceTooltipText(attendance) // undefined if no tooltip
...
                          <Tooltip
                            key={i}
                            tooltip={attendanceTooltip}
                            data-qa="attendance-tooltip"
                          >
                            <AttendanceCell>
                              <AttendanceTime data-qa="arrival-time">
                                {renderTime(attendance.arrived, date)}
                              </AttendanceTime>
                              <AttendanceTime data-qa="departure-time">
                                {renderTime(attendance.departed, date)}
                                {attendanceTooltip ? ' (*)'}
                              </AttendanceTime>
                            </AttendanceCell>
                          </Tooltip>
```